src/libsync/configfile.h

```diff
@@ -244,6 +245,7 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
 
     static QString _confDir;
     static QString _discoveredLegacyConfigPath;
+    bool _isComponentVisible;
```

Comment:
Please set primitive data members to a default value to avoid undefined behaviour

```suggestion
    bool _isComponentVisible = true;
```

Reply:
Done